cookbooks/cplanet/templates/default/update-planet.erb

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# DO NOT EDIT - This file is being maintained by Chef
+
+# setup
+
+PLANETDIR="<%= @basedir %>/planet"
+PLANETPREV="${PLANETDIR}/planet-previous.pbf"
+PLANETCURR="${PLANETDIR}/planet.pbf"
+PLANETNEW="${PLANETDIR}/planet-new.pbf"
+
+retval=1
+while [ $retval -eq 1 ]; do
+    pyosmium-up-to-date -v -o $PLANETNEW $PLANETCURR
+    retval=$?
+done
```

Comment:
So I discovered an interesting issue with this loop today while spinning up a new taginfo (which uses the same logic) from scratch.

The problem is that if planet is old enough that there is more than 1Gb of changes then `pyosmium-up-to-date` will stop after 1Gb but will exit one causing us to go round the loop and try again - except that `planet-new` hasn't been renamed over `planet` to it just tries to do the same thing again, but fails as `planet-new` already exists...